src/main/java/frc/robot/subsystems/elbow/ElbowState.java

```diff
@@ -8,7 +8,9 @@ public enum ElbowState {
 	MANUAL(new Rotation2d()),
```

Comment:
why not `Rotation2d.fromDegress(0)`? it is more consistent

Reply:
no need, also like that its better because in usage u dont really use the rot2d in these 2 states. so its look different here